xrpl4j-core/src/main/java/org/xrpl/xrpl4j/model/client/ledger/LedgerEntryRequestParams.java

```diff
@@ -402,6 +425,23 @@ default boolean binary() {
    */
   Optional<TicketLedgerEntryParams> ticket();
 
+  /**
+   * Look up a {@link org.xrpl.xrpl4j.model.ledger.BridgeObject} by {@link Address}. The {@link #bridge()} field must
```

Comment:
Since `BridgeObject` is only used in `LedgerEntryRequestParams`' static methods, the import does not carry over to the generated immutable, and the javadoc plugin errors. I typically try to shorten doc imports and then go through once I'm ready to open a PR and fix all the javadoc plugin errors by including the fully qualified class for any references that couldn't be found.